encryption/kyber/kyber.yaml

```diff
@@ -0,0 +1,20 @@
+name: CRYSTALS Kyber
+year: 2017
+category: lattice
+authors:
+  - Roberto Avanzi
+  - Joppe Bos
+  - Léo Ducas
+  - Eike Kiltz
+  - Tancrède Lepoint
+  - Vadim Lyubashevsky
+  - John M. Schanck
+  - Peter Schwabe
+  - Gregor Seiler
+  - Damien Stehlé
+reductions: |
+    M-LWE (tight ROM reduction, no-tight QROM reduction)
+state: none
+nist round: 2
+sources:
+  - NIST Submission Paper
```

Comment:
Having links to the papers and submissions would be nice. Or maybe dumping them in here as well and linking to them locally would be even better.

Reply:
As far as I know there's only the ZIP files on NIST's page, where the papers themselves cannot be linked directly

Reply:
We can use the scheme websites.
https://pq-crystals.org/kyber/data/kyber-specification-round2.pdf